lib/observer/observer.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Observer do
+require Satellite.Constants
+  defstruct [:latitude, :longitude, :height]
+
+  def create_from(latitude_degrees, longitude_degrees, height_km) do
```

Comment:
I was thinking that all the cases in the code where we do `* Constants.rad2deg` and `* Constants.deg2rad` should probably be a macro. Could even live in the `Constants` module. Something like:

```
defmacro deg2rad(deg), do: rad * pi() / 180
defmacro rad2deg(deg), do: rad * pi() / 180
```

Then we can have more fun with the pipe operator. I understand that it's probably just a holdover from `satellite-js`